### ISSUE TITLE
Dupe bug

### ISSUE THREAD

**Issue by ghost** [bafomdad/realfilingcabinet#54]
https://github.com/Lothrazar/Storage-Network/issues/127

**Comment by remclave** [comment]
I have actually confirmed it is a dupe bug that is occuring when there is interaction between **any storage bus system** and Real Filing Cabinet. It's even weirder and worse than that. 

Place a Filing Cabinet with one or more folders with ZERO items each. For testing, use an EMPTY ae2 system. Connect an ME Export Bus with two Capacity Cards to the Filing Cabinet. Load the filter with the items that will be allowed into the Filing Cabinet. **Now** link the ae2 system to the ME Export Bus.

The Filing Cabinet contents will fill with the filter items despite ZERO resources being available.

FOLDERS EMPTY
![2018-12-19_12 29 17](https://user-images.githubusercontent.com/31051239/50240800-5dc46400-038b-11e9-9265-316545b52c9b.png)

ME SYSTEM EMPTY
![2018-12-19_12 29 29](https://user-images.githubusercontent.com/31051239/50240814-66b53580-038b-11e9-8dfe-a7aa8da4fe06.png)

FILTER LOADED (and you can see ME System is not yet connected.)
![2018-12-19_12 29 37](https://user-images.githubusercontent.com/31051239/50240818-6a48bc80-038b-11e9-936e-e73ae19fcb94.png)

ME System connected and quantities in the folders are increasing.
![2018-12-19_12 29 59](https://user-images.githubusercontent.com/31051239/50241150-5d789880-038c-11e9-8500-a3bd84c97773.png)

**Comment by bafomdad** [comment]
yeah, it's a pretty bad thing to come across..I'm thinking it's making assumptions based on the item capability system that there /are/ items available to take in

**Comment by remclave** [comment]
I'm really hoping you can fix the cause since I actually LIKE your mod. I like it primarily for bulk storage.

**Comment by bafomdad** [comment]
version 0.1.79 will be out that should fix this issue. I am both relieved and horrified that it was such an easy mistake to find

**Comment by remclave** [comment]
It wasn't so easy to find. I saw a similar issue with a different mod (the one in the initial post for this thread) that also used a storage bus and didn't think it was an issue with your mod until I was testing while building a quest book for another mod pack. I don't know what prompted me to try without preloading resources in the AE2 system. But I'm glad I did. **Thank you so very much for finding the issue and fixing it so quickly.** Having players take advantage of such an ....interesting... dupe and not reporting it disturbs my sensibilities of fair play.
